llmfoundry/models/layers/attention.py

```diff
@@ -33,6 +33,13 @@ def is_flash_v1_installed():
     return version.parse(flash_attn.__version__) < version.parse('2.0.0')
 
 
+if is_flash_v1_installed():
```

Comment:
leave a comment behind about why this is necessary

Reply:
Done.